src/flows/recommendation_api/curated_corpus_candidates_flow.py

```diff
@@ -24,6 +24,7 @@
     TOPIC
 FROM "APPROVED_CORPUS_ITEMS"
 WHERE LANGUAGE = %(language)s
+AND IS_SYNDICATED = TRUE
```

Comment:
btw, this will more or less restrict things to `scheduled_surface_id=NEW_TAB_EN_US` as we had discussed in the previous version of the query.  i still think there may need to be a conversation about whether we want to query `SCHEDULED_CORPUS_ITEMS` in reverse chronological order if we want to take advantage of how Corrie batches the syndicated items.

Reply:
Feel free to create a PR to make changes to this query. Might be good to do as part of analyzing this corpus set in [HS-35](https://getpocket.atlassian.net/browse/HS-35).